src/applications/caregivers/containers/IntroductionPage.jsx

```diff
@@ -88,73 +108,92 @@ const IntroductionPage = ({ route, router }) => {
           <section className="process-step list-two">
             <h3 className="vads-u-font-size--h4">Apply</h3>
             <p>
-              Complete this Caregiver benefits form or have a Power of
-              Attorney/Representative complete form. You’ll get a confirmation
-              message after submitting the form. You can print this for your
-              records.
+              Each applicant needs to complete all form fields before submitting
+              the form. Once the form is submitted, you’ll receive a
+              confirmation message. You can print this for your records.
             </p>
 
-            <div className="vads-u-margin-bottom--2p5">
-              <PowerOfAttorneyInfo />
-            </div>
-            <RepresentativeInfo />
+            <p>
+              <b>Note:</b> If the Veteran isn’t enrolled in VA health care or is
+              currently on active duty with a medical discharge, they’ll need to
+              fill out an
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://www.va.gov/health-care/how-to-apply/"
```

Comment:
Another internal link

Reply:
So I removed all of these them the linter was complaining about not having it with `target="_blank"` so I went ahead and added it back so I could commit.